input/fsh/moped-procedure.fsh

```diff
@@ -3,6 +3,7 @@ Parent: Procedure
 Description: "MOPED Profil der Procedure Ressource für die LDF-Abrechnung"
 Title: "MOPED Procedure"
 
+* extension contains Leistungsanzahl named Leistungsanzahl 1..1
```

Comment:
das war Mal eine Diskussion, wo die Anzahl eingefügt werden soll. ist das der Vorschlag oder wurde das entschieden oder haben wir dafür noch eine offene Frage? falls noch nicht entschieden, sollten wir es zu den offenen Punkten aufnehmen. ich finde gerade auch keine elegantere lösung, aber vl. sollten wir das nochmal durchbesprechen um sicher zu sein, dass das hier die richtige stelle ist

Reply:
das ist mit der änderun in procedure nun nicht mehr relevant oder?